### ISSUE TITLE
NoneType fields not handled correctly for liabilities and throwing ApiTypeError

### ISSUE THREAD

**Issue by ipam73** [plaid/plaid-python#400]
We recently updated to plaid-python 8.10 from 7, and keep getting ApiTypeError for fields that are not being properly handled when None. This is breaking the experience for a number of our users (especially since this used to be handled properly with plaid-python 7).

The errors we're seeing are for the following fields:
1. last_payment_amount
2. last_statement_issue_date

Errors:

> ApiTypeError("Invalid type for variable 'last_payment_amount'. Required value type is float and passed type was NoneType at ['received_data']['liabilities']['credit'][1]['last_payment_amount']")
> 
> ApiTypeError("Invalid type for variable 'last_statement_issue_date'. Required value type is date and passed type was NoneType at ['received_data']['liabilities']['credit'][0]['last_statement_issue_date']")
> 

What we're doing:
```
 request = LiabilitiesGetRequest(access_token=access_token)
 response = client.liabilities_get(request)
```

**Comment by phoenixy1** [comment]
Very sorry for the trouble! I just submitted a PR to fix this and it should be incorporated into next week's schedule client library release.

**Comment by ipam73** [comment]
Thank you! I'll update as soon as it's released

**Comment by phoenixy1** [comment]
@ipam73 ok so I have a slight correction to my previous response! 

Originally I thought that these fields were erroneously not marked `nullable` in our API spec and client libraries (this has happened in the past) and I submitted a fix for that. However, it turns out the actual problem is that these fields in fact aren't supposed to be nullable, so marking the fields as `nullable` isn't really the right way to fix it.

I have raised an issue with the team that owns credit card liabilities data and they are investigating the root issue and working a on fix. Would it work for you/your team to catch the error in the meantime?
